var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -79,9 +79,7 @@ class Blaspp(CMakePackage, CudaPackage, ROCmPackage):
 
     requires("%oneapi", when="+sycl", msg="blaspp+sycl must be compiled with %oneapi")
 
-    patch(
-        "0001-fix-blaspp-build-error-with-rocm-6.0.0.patch", when="@2023.06.00: ^hip@6.0.0 +rocm"
-    )
+    patch("0001-fix-blaspp-build-error-with-rocm-6.0.0.patch", when="@2023.06.00: ^hip@6.0 +rocm")
```

Comment:
(follow-up) I'm guessing this is because it also applies for `hip@6.0.2` - that works fine although the patch name is no longer accurate